.travis.yml

```diff
@@ -1,26 +1,34 @@
-dist: trusty
 language: node_js
 node_js:
-  - 8
-  
+  - '8'
+
 os:
-  - osx
   - linux
+  - osx
+
+notifications:
+  email: true
```

Comment:
It appears that putting "email: true" doesn't actually send out notifications. From the [travis documentation](https://docs.travis-ci.com/user/notifications/#configuring-email-notifications) you have to specify the email addresses that you want notifications to go to as follows:

```
notifications:
  email:
    - one@example.com
    - other@example.com
```

Using ```email: false``` is what is used to indicate that no email notifications are desired.

Reply:
If you completely remove the `notifications -> email` settings, it will just send notifications. So if `true` doesn't work, but emails should be sent to the same people as before, then just removing these two lines does also work.

I actually added this and set it to `false`, because I didn't want to bother everyone while testing Travis settings. Not sure, if Travis website has default settings for notifications.